javaparser-core/src/main/java/com/github/javaparser/ast/ArrayCreationLevel.java

```diff
@@ -128,12 +128,6 @@ public ArrayCreationLevel setAnnotations(final NodeList<AnnotationExpr> annotati
         return this;
     }
 
-    @Override
-    @Generated("com.github.javaparser.generator.core.node.GetNodeListsGenerator")
-    public List<NodeList<?>> getNodeLists() {
```

Comment:
Are we removing this public method? Could it be useful to someone? Maybe we could implement it once in Node using the metamodel

Reply:
The reason I'm removing it is because I can't imagine it being useful to anyone - do you have use cases? If so, it would be nice to at least rename it, I remember people being confused over what it does.

Reply:
Well I guess we can always remove it and put it back if people complain. Anyway for "smart" usages the result can be obtained from the metamodel I guess